plugins/scaffolder-backend/src/scaffolder/stages/publish/github.ts

```diff
@@ -44,13 +44,14 @@ export class GithubPublisher implements PublisherBase {
     values: RequiredTemplateValues & Record<string, JsonValue>,
   ) {
     const [owner, name] = values.storePath.split('/');
+    const description = String(values.description);
```

Comment:
Looks good, the only thing I could think would be to apply any validations like field length but there's none documented on the github api and it's sent in post body (so no need for any url encoding ).